fvm/bootstrap.go

```diff
@@ -55,6 +55,14 @@ type BootstrapProcedure struct {
 	identities flow.IdentityList
 }
 
+type BootstrapAccountKeys struct {
```

Comment:
This structure can be private right?

Reply:
I don't think so because it is used in the option
```
func WithBootstrapAccountKeys(keys BootstrapAccountKeys) BootstrapProcedureOption {
```

I could rewrite the option so you have to pass keys individually. Would that be preferred?

Reply:
No it is fine with the public struct. I just thought it's not used anywhere else :)